### DISCUSSION TITLE
Make pagination options controllable

### DISCUSSION THREAD

**Discussion by ionutbaltariu** [KevinVandy/material-react-table#316] [Q&A] [answered]
The table could include props that would allow the change of the default items per page or the available options regarding the items per page (with a maximum of 100, dictated by material-ui).

**Comment by KevinVandy** [comment, marked as answer]
It does. Everything is customizable. https://www.material-react-table.com/docs/guides/pagination#customize-pagination-components

**Reply by ionutbaltariu** [reply]
that's awesome! thanks!
